packages/react/src/components/Radio/Radio.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Radio } from './Radio';
+
+describe('Radio', () => {
```

Comment:
Should we add one test for the description as well? I suggest checking that the description prop works and that the input field gets the aria-describedBy attribute when the description is provided. Consider checking if aria-describedBy matches the ID for the description element. 🤔

Reply:
Added tests for this